File: Laravel Server/app/Models/Order.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Order extends Model
{
    use HasFactory;
    protected $fillable = [
        'user_id',
        "product_name",
        "feedback",
        'product_id', 'number',
        "total", "status"
    ];
    public function Product()
    {
        return $this->belongsTo('App\Models\Product', 'id', 'product_id');
    }
    public function User()
    {
        return $this->belongsTo('App\Models\User', 'id', 'user_id');
    }
}

File: Application/lib/classes/order.dart
class Order {
  String id;
  String productId;
  double total;
  int number;
  String productName;
  String status;
  String feedback;
  Order(
      {required String this.productId,
      required String this.id,
      required double this.total,
      required String this.productName,
      String this.status = "pending",
      required int this.number,
      String this.feedback = ""});
}
